Sources/Testing/Running/Configuration.TestFilter.swift

```diff
@@ -365,7 +428,7 @@ extension Configuration.TestFilter {
     case or
 
     /// The equivalent of this instance as a callable function.
```

Comment:
Nitpick: add more markup here now that it's a function.

Reply:
Oh, I can enhance it to use `callAsFunction()`!